### DISCUSSION TITLE
我的网站部署失败

### DISCUSSION THREAD

**Discussion by XS170202102296** [imsyy/home#267] [Q&A] [answered]
### 站点链接

https://fctestweb.rth5.com/

### 问题描述

我把源码上传之后没有正确显示内容

### 补充信息

我之前在b战找过你但是你平时很少看b战没回我，我才从GitHub找你的，看到请回我谢谢啦，我之前还从某个网站上面买到你这个源码，买了才发现你开源的从里面的说明文档里面看到，然后我找到你GitHub，看到版本已经迭代了这么多了就想升级一下，上传之后就跟那个卖源码的不一样，之前我买的那个源码（就是你的旧版本那个）直接上传就能打开网站的，但是你这个GitHub上传源码之后就没有正常显示，本人略懂html和JavaScript还有css，vue不是很懂，但是也会丢丢，作者看到请帮一下我吧，谢谢啦

**Comment by imsyy** [comment, marked as answer]
抱歉，最近比较忙，如果部署不了的话，可以看下这里：https://github.com/imsyy/home?tab=readme-ov-file#%E8%87%AA%E5%8A%A8%E9%83%A8%E7%BD%B2

**Reply by XS170202102296** [reply]
但是我有自己的主机，不需要用到GitHub，因为GitHub访问速度实在感人

**Comment by imsyy** [comment]
> 但是我有自己的主机，不需要用到GitHub，因为GitHub访问速度实在感人

有自己的服务器可以用 Docker 部署

**Reply by XS170202102296** [reply]
那如果是虚拟主机呢？

**Reply by xiaobaimc** [reply]
虚拟主机可以把构建生成的静态资源上传上去，或者去releases下载[3.1.3-fix](https://github.com/imsyy/home/releases/download/v3.1.3/3.1.3-fix.zip)，这个是静态版本的

**Reply by XS170202102296** [reply]
好的，我试试👌(｡•̀ᴗ-)✧
